accio-tests/src/test/java/io/accio/testing/bigquery/TestReloadCache.java

```diff
@@ -95,22 +98,24 @@ public void testReloadCache()
         assertThat(taskInfo.getEndTime()).isNotNull();
         assertThat(taskInfo.getEndTime()).isAfter(taskInfo.getStartTime());
 
-        CachedTable cachedTable = taskInfo.getCachedTables().get(0);
+        CachedTable cachedTable = taskInfo.getCachedTable();
         assertThat(cachedTable.getErrorMessage()).isEmpty();
         assertThat(cachedTable.getName()).isEqualTo("Revenue");
         assertThat(cachedTable.getRefreshTime()).isEqualTo(Duration.valueOf("5m"));
         assertThat(cachedTable.getCreateDate()).isNotNull();
 
         rewriteFile("cache/cache_reload_3_mdl.json");
         reloadAccioMDL();
-        taskInfo = reloadCacheAsync();
-        assertThat(taskInfo.getTaskStatus()).isEqualTo(RUNNING);
-        taskInfo = waitUntilFinished(taskInfo.getTaskId());
-        cachedTable = taskInfo.getCachedTables().get(0);
+        List<TaskInfo> taskInfos3 = reloadCacheAsync();
+        assertThat(taskInfos3.size()).isEqualTo(1);
+        taskInfo = taskInfos3.get(0);
+        // assertThat(taskInfo.getTaskStatus()).isEqualTo(RUNNING);
```

Comment:
Why do we comment out this line ?

Reply:
Just forgot to change it back after fixing test.